src/longtask.js

```diff
@@ -79,7 +79,7 @@ export class SplunkLongTaskInstrumentation extends InstrumentationBase {
 
   _isSupported() {
     // note: PerformanceObserver.supportedEntryTypes has better browser support than LongTask
-    const supportedEntryTypes = PerformanceObserver && PerformanceObserver.supportedEntryTypes;
+    const supportedEntryTypes = window.PerformanceObserver && window.PerformanceObserver.supportedEntryTypes;
```

Comment:
second `window` is redundant as first confirms it's available as global variable
```suggestion
    const supportedEntryTypes = window.PerformanceObserver && PerformanceObserver.supportedEntryTypes;
```